app/main/helpers.js

```diff
@@ -3,6 +3,8 @@ import settings from '../shared/settings';
 import i18n from '../configs/i18next.config';
 import { makeOpenBrowserWindow, makeSetSavedEnv } from '../../gui-common/util';
 
```

Comment:
Should avoid importing this module from shared places for browser/electron? (The L1's electron reference)

Reply:
I don't know. Wasn't this here already though?

Reply:
Maybe no files loaded this file...?
Just grepped current main branch:
```
(headspin) kazu$ git grep 'helpers'
app/renderer/lib/appium-client.js:import {getWebviewStatusAddressBarHeight, parseSource, setHtmlElementAttributes} from './webview-helpers';
package-lock.json:        "@babel/helpers": "^7.17.2",
package-lock.json:    "node_modules/@babel/helpers": {
package-lock.json:      "resolved": "https://registry.npmjs.org/@babel/helpers/-/helpers-7.17.2.tgz",
package-lock.json:        "@babel/helpers": "^7.17.2",
package-lock.json:    "@babel/helpers": {
package-lock.json:      "resolved": "https://registry.npmjs.org/@babel/helpers/-/helpers-7.17.2.tgz",
test/e2e/pre-e2e.test.js:import { e2eBefore, e2eAfter } from '../../gui-common/test-helpers';
test/unit/webview-helpers-specs.js:import {parseSource} from '../../app/renderer/lib/webview-helpers';
test/unit/webview-helpers-specs.js:describe('webview-helpers.js', function () {
```

Reply:
I don't think I added this block of code. I think it as already there.

Reply:
i mean probably like https://github.com/appium/appium-inspector/pull/371/files#diff-cc0a8b1e7efd7e5b9ee01a00c256283251315b2c0c18cca08911fef1c01fa0b9R16 is the first time to load this file. No dependencies to load this file existed before, so "already there" is correct but no references were there.

```
index.js:8 Uncaught TypeError: fs.existsSync is not a function
    at getElectronPath (index.js:8:10)
    at Object.parcelRequire.../../node_modules/electron/index.js.fs (index.js:21:18)
    at newRequire (renderer.e31bb0bc.js:47:24)
    at localRequire (renderer.e31bb0bc.js:53:14)
    at Object.parcelRequire.../main/helpers.js.electron (helpers.js:1:1)
    at newRequire (renderer.e31bb0bc.js:47:24)
    at localRequire (renderer.e31bb0bc.js:53:14)
    at Object.parcelRequire.actions/Session.js.../../shared/settings (Session.js:16:1)
    at newRequire (renderer.e31bb0bc.js:47:24)
    at localRequire (renderer.e31bb0bc.js:53:14)
    at Object.parcelRequire.containers/SessionPage.js.redux (SessionPage.js:4:1)
    at newRequire (renderer.e31bb0bc.js:47:24)
    at localRequire (renderer.e31bb0bc.js:53:14)
    at Object.parcelRequire.routes.js.react (routes.js:4:1)
    at newRequire (renderer.e31bb0bc.js:47:24)
    at localRequire (renderer.e31bb0bc.js:53:14)
```